styles/_base.css

```diff
@@ -23,6 +23,10 @@ h2 {
   font-size: 20px;
 }
 
+main {
+  overflow: scroll;
+}
+
 input,
```

Comment:
Do you have an `input`-element in your HTML? If not, then you don't need to style it 🙂